tests/src/test/resources/application.conf

```diff
@@ -1,6 +1,6 @@
 tests {
-    delta-uri = "http://localhost:8080/v1"
-    realm-uri = "http://"${"keycloak:8080"}"/auth/realms"
```

Comment:
this is required if you want to do local debugging

Reply:
Resolver creation was failing because of this as it wasn't able to fetch the openId config

Reply:
that's very strange; the docker compose plugin sets the `keycloak:8080` env variable.

Reply:
Yep, but we use the realm-uri for realm payload for the config url so the `keycloak:8080` env variable resolves to localhost:xxxx which can't be reached by the delta instance as it is the address exposed outside the docker compose network. Inside it is `keycloak:8080`